notebooks/demo.ipynb

```diff
@@ -455,6 +449,515 @@
+      ],
+      "text/plain": [
+       "<IPython.core.display.Markdown object>"
+      ]
+     },
+     "metadata": {},
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "from IPython.display import Markdown, display\n",
+    "\n",
+    "display(Markdown(\"\"\"\n",
+    "Lorem ipsum dolor sit amet **{}**, consectetur adipiscing elit **{}**, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum.\n",
+    "\"\"\".format(3.14, 6.28)))"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "#### Subsubsection"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 23,
+   "metadata": {
+    "jupyter": {
+     "source_hidden": true
+    }
+   },
+   "outputs": [
+    {
+     "data": {
+      "text/markdown": [
+       "\n",
+       "Lorem ipsum dolor sit amet **3.14**, consectetur adipiscing elit **6.28**, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum.\n"
+      ],
+      "text/plain": [
+       "<IPython.core.display.Markdown object>"
+      ]
+     },
+     "metadata": {},
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "from IPython.display import Markdown, display\n",
+    "\n",
+    "display(Markdown(\"\"\"\n",
+    "Lorem ipsum dolor sit amet **{}**, consectetur adipiscing elit **{}**, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum.\n",
+    "\"\"\".format(3.14, 6.28)))"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Notes\n",
+    "\n",
+    "-   Markdown cells containing multiple \"sections\" only show as the first section in the ToC. Accordingly, users should add a **new** Markdown cell for **each** section.\n",
+    "-   The ToC extension **does** detect multiple \"sections\" when those sections are generated by code cells. However, users are likely to run into buggy ToC behavior when doing so. Users are strongly encouraged to **avoid** using code cells to generate section headings.\n",
```

Comment:
I'd rather not advertise vague buggy behavior here. If it is indeed a bug, let's briefly describe the current behavior, say why it's a problem, then open an issue on the repo.

Reply:
I updated the note to include an example. I am not convinced it's a issue worthy bug rather than a user "error" in doing things which are generally not recommended, but are allowed within Jupyter.